src/lib/litegraph/src/LGraphCanvas.ts

```diff
@@ -8031,7 +8031,13 @@ export class LGraphCanvas
           has_submenu: true,
           callback: LGraphCanvas.onMenuAdd
         },
-        { content: 'Add Group', callback: LGraphCanvas.onGroupAdd }
+        { content: 'Add Group', callback: LGraphCanvas.onGroupAdd },
+        {
+          content: 'Paste',
```

Comment:
Are these internationalized by default?

Reply:
Yes, it should be, but right now our translation workflow is down